src/cljs/athens/views/node_page.cljs

```diff
@@ -201,6 +212,14 @@
         :class      (when (= editing-uid uid) "is-editing")
         :auto-focus true
         :on-change  (fn [e] (db-handler (.. e -target -value) uid))}])
+    [button {:on-click-fn (fn [e]
+                            (doall (swap! show-page-menu? not)
+                                   (reset! page-menu-position {:x (.. e -target getBoundingClientRect -left) :y (.. e -target getBoundingClientRect -bottom)})))
```

Comment:
I would make a `let` binding for bounding client rect. This makes it easier to acces `-left` and `-bottom`.

I don't think the `doall` is necessary.